packages/plugins/dart/flutter-freezed/src/index.ts

```diff
@@ -1,38 +1,39 @@
 import { oldVisit, PluginFunction, Types } from '@graphql-codegen/plugin-helpers';
 import { transformSchemaAST } from '@graphql-codegen/schema-ast';
 import { GraphQLSchema } from 'graphql';
-import { FlutterFreezedPluginConfig } from './config.js';
-import { FreezedDeclarationBlock } from './freezed-declaration-blocks/index.js';
-import { schemaVisitor } from './schema-visitor.js';
-import { addFreezedImportStatements, DefaultFreezedPluginConfig } from './utils.js';
+import { FlutterFreezedPluginConfig } from './config';
+import { schemaVisitor } from './schema-visitor';
+import { buildImportStatements, defaultFreezedPluginConfig } from './utils';
```

Comment:
@Parables all relative imports should end with a `.js` (for ESM support)

This is what is causing the CI tests to fail

Reply:
Thanks very much for the feedback